publications/LCS3.tex

```diff
@@ -0,0 +1,20 @@
+\section{LCS 3 - LCS Review Committee Members
+(2019-11-02)}\label{lcs-3---lcs-review-committee-members-2019-11-02}
+
+Publication Date: 2019-11-02
+
+\subsection{§1 Forward}\label{forward}
+
+This is an informative document which defines the Members of the LCS
+Review Committee, a group which is defined by LCS 1 and LCS 2.
+
+This is a special LCS Document in the same standing as LCS 1 and LCS 2.
+It shall be subjected to annual Committee Review as defined in §2.3 of
+LCS 2.
+
+\subsection{§2 LCS Review Committee
+Members}\label{lcs-review-committee-members}
+
+The following persons have been appointed as members of the LCS Review
+Committee and are voluntarily serving as such: * Connor Horman * Matthew
+Johnson * Maelen Kibler * Ray Redondo
```

Comment:
I don't think this got formatted correctly.

Reply:
You're right. It did not. *Proceeds to yeet pandoc*
I should probably learn LaTex